dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -65,7 +65,7 @@ public ItemRest findOne(Context context, UUID id) {
         List<Item> items = new ArrayList<Item>();
         int total = 0;
         try {
-            total = is.countTotal(context);
+            total = is.countArchivedNotWithdrawnItems(context);
```

Comment:
It is good to have confirmation that the cause of this issue is the withdrawn items.

In an admin context the method findAllUnfiltered() returns withdrawn items.  It would be good to confirm that the count is accurate.

Reply:
The class ItemService offers 2 methods countItems() and countAllItems().  

This class has methods findAll() and findAllUnfiltered().

Perhaps the count method names should match one of those conventions.

Reply:
In the context of the ItemService, this seems OK.  I don't have a strong opinion, but I noticed ItemService also has  `countNotArchivedItems()`, `countWithdrawnItems()`.... so `countArchivedNotWithdrawnItems()` seems to be a similar naming convention.